frontend/components/tableDisplay/petTableDisplay.tsx

```diff
@@ -0,0 +1,126 @@
+import { Table, Input, Button, Space } from 'antd';
+import { PlusOutlined, ImportOutlined } from '@ant-design/icons';
+import SwitchTag from './switchTag';
+import TableName from './tableNameColumn';
+import React, { useState } from 'react';
+
+const PetTableDisplay = () => {
+  const [searchText, setSearchText] = useState("");
+  const [searchedColumn, setSearchedColumn] = useState("");
+
+  const { Search } = Input;
+  const onSearch = () => { };
+
+
+  const columns = [
+    {
+      title: 'ID',
+      dataIndex: 'key',
+      defaultSortOrder: 'ascend',
+      sorter: (a, b) => a.key - b.key,
+    },
+    {
+      title: 'Name',
+      dataIndex: 'name',
+      sorter: (a, b) => a.name.localeCompare(b.name),
+      render: (name, record) => <TableName name={name} image={record.image} />,
+    },
+    {
+      title: 'Visibility',
+      dataIndex: 'visibility',
+      onFilter: (value, record) => record.visibility.indexOf(value) === 0,
```

Comment:
can explain the rationale to why we are checking the equality this way?

Reply:
This is the method in the antd documentation to create a sorting system in the column, but happy to change if you think there's a better way! See https://codesandbox.io/s/1kdbt line 60